fuse/file.go

```diff
@@ -78,3 +94,13 @@ func (fh fileHandle) Read(ctx context.Context, req *fuse.ReadRequest, resp *fuse
 	resp.Data = buf[:n]
 	return err
 }
+
+func (fh fileHandle) Write(ctx context.Context, req *fuse.WriteRequest, resp *fuse.WriteResponse) error {
+	// The Write interface doesn't allow returning partial writes because they never occur for remote writes.
+	err := fh.w.Write(ctx, req.Offset, req.Data)
```

Comment:
Would it hurt to return the # of bytes written? If it's easy, we might as well keep the symmetry with `io.Writer`. People may want to use Wash for non-remote stuff too.